objects/src/accounts/seed.rs

```diff
@@ -0,0 +1,65 @@
+use super::{parse_felt, AccountError, AccountId, Hasher, Word};
+
+// ACCOUNT SEED GENERATOR
+// ================================================================================================
+
+/// Specifies the account type.
+pub(super) enum AccountType {
+    FungibleFaucet,
+    NonFungibleFaucet,
+    RegularAccountImmutableCode,
+    RegularAccountUpdatableCode,
+}
+
+/// Finds and returns a seed suitable for creating account IDs for the specified account type
+pub(super) fn get_seed(
+    init_seed: [u8; 32],
+    account_type: AccountType,
+    on_chain: bool,
+) -> Result<Word, AccountError> {
+    let mut current_seed: Word = [
+        parse_felt(&init_seed[..8])?,
+        parse_felt(&init_seed[8..16])?,
+        parse_felt(&init_seed[16..24])?,
+        parse_felt(&init_seed[24..32])?,
+    ];
```

Comment:
I wonder if we need to check for valid field elements here? Maybe doing silent conversions to field elements is enough?